src/libraries/Native/Unix/System.Globalization.Native/pal_icushim.h

```diff
@@ -8,3 +8,11 @@ PALEXPORT int32_t GlobalizationNative_LoadICU(void);
 PALEXPORT void GlobalizationNative_InitICUFunctions(void* icuuc, void* icuin, const char* version, const char* suffix);
 
 PALEXPORT int32_t GlobalizationNative_GetICUVersion(void);
+
+#if defined(STATIC_ICU)
+
+PALEXPORT int32_t GlobalizationNative_LoadICUData(void * pData);
+
+PALEXPORT const char* GlobalizationNative_GetICUDTName(const char* culture);
```

Comment:
Do we need this to be exported? It seems like it is only used by `pal_icushim_static.c`

Reply:
It's used here too https://github.com/steveisok/runtime/blob/2a5ad94802fdbf5af599dad69d93e5f7d8615ec2/src/tasks/AppleAppBuilder/Templates/runtime.m#L249

Reply:
I was referring to: `GlobalizationNative_GetICUDTName`? At least my CTRL+F search in your PR is not showing me any other hit so I might be missing something 😄

Reply:
Ah, I'd like to keep it in for now.  It may be useful depending on how we support different data configurations.

Reply:
sounds good!